examples/advanced/gnn-pt/jobs/gnn-pt/app/config/config_fed_server.json

```diff
@@ -0,0 +1,59 @@
+{
+  "format_version": 2,
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "components": [
+    {
+      "id": "persistor",
+      "name": "PTFileModelPersistor",
+      "args": {
+        "model": {
+          "path": "torch_geometric.nn.GraphSAGE",
+          "args": {
+            "in_channels": 50,
+            "hidden_channels": 64,
+            "num_layers": 2,
+            "out_channels": 64
+            
+          } 
+        }
+      }
+    },
+    {
+      "id": "shareable_generator",
+      "path": "nvflare.app_common.shareablegenerators.full_model_shareable_generator.FullModelShareableGenerator",
+      "args": {}
+    },
+    {
+      "id": "aggregator",
+      "path": "nvflare.app_common.aggregators.intime_accumulate_model_aggregator.InTimeAccumulateWeightedAggregator",
+      "args": {
+        "expected_data_kind": "WEIGHTS"
+      }
+    },
+    {
+      "id": "model_selector",
+      "name": "IntimeModelSelector",
+      "args": {
+        "key_metric": "test_f1"
```

Comment:
This key_metric should match the key of the metric in the returned FLModel.
That's why I ask to change the line over there to "test_f1"